src/main/java/svc/models/Court.java

```diff
@@ -1,15 +1,22 @@
 package svc.models;
 
 import java.math.BigDecimal;
+import java.util.List;
 
 public class Court 
 {
 	public int id;
-	public BigDecimal latitude;
-	public BigDecimal longitude;
-	public String municipality;
+	public String court_name;
+	public String phone;
+	public String website;
+	public String extension;
 	public String address;
+	public String payment_system;
 	public String city;
 	public String state;
 	public String zip_code;
+	public BigDecimal latitude;
+	public BigDecimal longitude;
+	
+	public List<MunicipalityJudge> municipality_judges;
```

Comment:
A Court can have many judges?

What about municipalities? Shouldn't there be a list of munis that are assigned to a Court?